src/components/panels/ConfigPanel/BooleanConfig.tsx

```diff
@@ -85,19 +108,25 @@ export default function BooleanConfig({
         {config.name}
       </h5>
       <button
-        className={classnames("u-float-right p-button--base", {
-          "u-hide": !showUseDefault,
-        })}
+        className={classnames(
+          "u-float-right p-button--base config-panel__hide-button",
+          {
+            "config-panel__show-button": showUseDefault,
+          }
+        )}
         onClick={resetToDefault}
       >
         use default
       </button>
       <div
         className={classnames("config-input--description", {
-          "u-hide": !showDescription,
+          "config-input--description__show": showDescription,
```

Comment:
Is this a good case for a [CSS state class](https://canonical-web-and-design.github.io/practices/coding/stylesheets.html#state-classes)

Reply:
Turns out this was actually supposed to be removed - in the conflict resolution I appear to have lost an entire commit. That stuff is back now and these lines no longer exist 👍 good catch!